dev-name.md

```diff
@@ -0,0 +1,5 @@
+# Developer Name
+
+Hi!
+
+My name is Abdullah.
```

Comment:
This's a good start. I would suggest adding more information about the developer, such as web development preference.
```suggestion
My name is Abdullah and I like web development.
```